streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractKeyValueStoreTest.java

```diff
@@ -383,13 +394,29 @@ public void shouldThrowNullPointerExceptionOnGetNullKey() {
     }
 
     @Test
-    public void shouldThrowNullPointerExceptionOnRangeNullFromKey() {
-        assertThrows(NullPointerException.class, () -> store.range(null, 2));
+    public void shouldReturnValueOnRangeNullFromKey() {
```

Comment:
While we're at it, should we add one to test `range(null, null)`?

Reply:
Yup, good point, added.